src/Horarium/Fallbacks/FallbackStrategyOptions.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using Horarium.Builders.Fallback;
+using Horarium.Builders.Parameterized;
+using Horarium.Interfaces;
+
+namespace Horarium.Fallbacks
+{
+    internal class FallbackStrategyOptions : IFallbackStrategyOptions
+    {
+        private readonly TimeSpan _globalObsoleteInterval;
+        
+        public FallbackStrategyTypeEnum? FallbackStrategyType { get; private set; }
+        public JobMetadata FallbackJobMetadata { get; private set; }
+
+        public FallbackStrategyOptions(TimeSpan globalObsoleteInterval)
+        {
+            _globalObsoleteInterval = globalObsoleteInterval;
+        }
+        
+        public void CreateFallbackJob<TJob, TJobParam>(TJobParam parameters, Action<IFallbackJobBuilder> fallbackJobConfigure = null) where TJob : IJob<TJobParam>
```

Comment:
There are some inconsistency between enum member `ScheduleFallbackJob` and method name `CreateFallbackJob`. Imo, `Schedule` is more meaningful verb here